### ISSUE TITLE
Don't warn of compatible vendor prefix in a vendor property

### ISSUE THREAD

**Issue by nschonni** [CSSLint/csslint#348]
An example from jQuery Mobile:

``` css
.ui-slider-input::-webkit-outer-spin-button,
.ui-slider-input::-webkit-inner-spin-button {
    -webkit-appearance: none;
    margin: 0;
}
```

This produces `The property -moz-appearance is compatible with -webkit-appearance and should be included as well. Include all compatible vendor prefixes to reach a wider range of users. (compatible-vendor-prefixes)` even though the `-moz-appearance` wouldn't make sense in a WebKit pseudo property.

**Comment by broonage** [comment]
+1

**Comment by ahmadajmi** [comment]
+1

**Comment by yastrebovb** [comment]
+1
